static/app/components/assigneeSelector.tsx

```diff
@@ -26,7 +26,7 @@ import space from 'sentry/styles/space';
 import {Actor, SuggestedOwner, SuggestedOwnerReason, Team, User} from 'sentry/types';
 import {buildTeamId, buildUserId, valueIsEqual} from 'sentry/utils';
 
-type SuggestedAssignee = Actor & {
+export type SuggestedAssignee = Actor & {
```

Comment:
I feel like maybe this could live in `sentry/types` somewhere?

I generally dislike exporting types from UI components like this. But that's somewhat of a gut feeling.

Reply:
Removed this change, it is no longer necessary due to updates to reuse `AssigneeSelector`!